guides/micronaut-jms-aws-sqs/java/src/main/java/example/micronaut/DemoController.java

```diff
@@ -0,0 +1,20 @@
+package example.micronaut;
+
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+
+@Controller  // <1>
+public class DemoController {
+
+    private final DemoProducer demoProducer;
+
+    public DemoController(DemoProducer demoProducer) {  // <2>
+        this.demoProducer = demoProducer;
+    }
+
+    @Get("/demo")  // <3>
+    public void publishDemoMessages() {
```

Comment:
The controller which has a side effect it publishes to a Queue. It should be mapped to POST instead of Get. 

I would annotate the method with `@Status(NO_CONENT)` and respond a 204 instead 200.

Reply:
Make sense. will change that.